### PR TITLE
fix: use api for latest runs instead of local filtering

### LINKED ISSUE #80: Quiver - bug: table view misses most entries
The table view misses most of its entries. This is due to the fact that the store used for this filters the runs manually by newest date. If a run happens to be running over multiple days, only the entries with a timestamp of the latest day get picked - the rest will be ignored. 

Solution: use the `/runs/latest` API endpoint instead of filtering manually by date.

### PR DESCRIPTION
**Pull request by jfrer**
closes #80

### CODE REVIEW

File: src/helpers/api.ts [hunk 1]
```diff
@@ -39,7 +39,7 @@ async function getLatestRuns(gtId?: string, workflowId?: string): Promise<Evalua
 
     path += '/latest'
 
-    return await request(path)
+    return (await request(path)).flat(1)
```
[the remark this answers is not in the document]
Comment (mweidling) [reply src/helpers/api.ts:42]: Yes, it seems to be faulty here on back hand side.

[the remark this answers is not in the document]
Comment (jfrer) [reply src/helpers/api.ts:42]: Should we wait for this to get fixed to change this line before merging?

[the remark this answers is not in the document]
Comment (t11r) [reply src/helpers/api.ts:42]: At least add `TODO` and a comment explaining the workaround.

[the remark this answers is not in the document]
Comment (jfrer) [reply src/helpers/api.ts:48]: Added a comment in 6cd3d17.

**Review by paulpestov — APPROVED** [review]

**Pull request merged**